edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Program.cs

```diff
@@ -254,6 +256,19 @@ static string GetStoragePath(IConfiguration configuration)
             return storagePath;
         }
 
+        static string GetLocalConfigFilePath(IConfiguration configuration, ILogger logger)
+        {
+            string localConfigPath = configuration.GetValue<string>("LocalConfigPath");
+
+            if (string.IsNullOrWhiteSpace(localConfigPath))
+            {
+                logger.LogInformation("No local config path specified. Using default path instead.");
```

Comment:
Nit: It would be good to log which config path is being used as well 
LogInformation("Using local config path .. ")

Reply:
Added a log for the path